istio-interop-testing/build.gradle

```diff
@@ -0,0 +1,68 @@
+plugins {
+    id "application"
+    id "java"
+
+    id "com.google.protobuf"
+    id 'com.google.cloud.tools.jib'  // For releasing to Docker Hub
```

Comment:
No need for the comment here and below. That was present in the example because it was an example and isn't normally needed.